src/main/java/com/shipmentEvents/demo/Testing.java

```diff
@@ -0,0 +1,46 @@
+package com.company;
+
+import java.util.concurrent.locks.*;
+
+public class Testing {
+
+    public static class Table {
+
+        private static Lock Flashlight = new ReentrantLock();
+        private static Lock Batteries = new ReentrantLock();
+
+        public static void giveFlashLightAndBatteries() {
+            try {
+                Flashlight.lock();
+                Batteries.lock();
+                System.out.println("Lights on");
+            } finally {
+                Batteries.unlock();
+                Flashlight.unlock();
+            }
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 13:20, 24:31.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.